src/index.js

```diff
@@ -111,15 +88,16 @@ function displayAllData() {
 }
 
 function updateUserDataDOM(userInfo) {
-  $(`<p>Welcome,</p><h1>${user.getFirstName()}</h1>`).prependTo(name);
-  address.text(userInfo.address);
-  email.text(userInfo.email);
-  strideLength.text(userInfo.strideLength);
-  dailyStepGoal.text(userInfo.dailyStepGoal.toLocaleString());
-  friendList.text(userRepository.getFriendsName().join(', '));
+  $(`<p>Welcome,</p><h1>${user.getFirstName()}</h1>`).prependTo($('#name'));
+  $('#address').text(userInfo.address);
+  $('#email').text(userInfo.email);
+  $('#strideLength').text(userInfo.strideLength);
+  $('#stepGoal').text(userInfo.dailyStepGoal.toLocaleString());
+  $('#step-compare').text(userRepository.getFriendsName().join(', '));
```

Comment:
Looking at these side-by-side really illustrates how these global variables weren't really saving any characters in the end.